fastlane/metadata/android/ka/title.txt

```diff
@@ -0,0 +1 @@
+აპლიკაციის მაღაზიის სათაური, ლიმიტია 30 სიმბოლო
```

Comment:
Translated to `Titre de l'App Store, limite de 30 caractères`, this is an error.